script.js

```diff
@@ -21,4 +21,176 @@ navListEl.forEach((element) => {
+    technologies: ['HTML', 'CSS', 'JavaScript'],
+    technologies1: ['github', 'ruby', 'Bootstap'],
+    liveVersion: 'https://gmanlove.github.io/My-Portfolio/ ',
+    sourceCode: 'https://github.com/Gmanlove/My-Portfolio',
+  },
+  {
+    name: 'Facebook 360',
+    des: ['Canopy', 'backend', '2015'],
+    img: './images/Snapshoot Portfolio (2).svg',
+    description:
+      'Personal portfolio that contains project information, skills, and contact',
+    desc1:
+      'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry.',
+    technologies: ['HTML', 'CSS', 'JavaScript'],
+    technologies1: ['github', 'ruby', 'Bootstrap'],
+    liveVersion: 'https://gmanlove.github.io/My-Portfolio/ ',
+    sourceCode: 'https://github.com/Gmanlove/My-Portfolio',
+  },
+  {
+    name: 'Uber Navigation',
+    des: ['Canopy', 'backend', '2015'],
+    img: './images/Snapshoot Portfolio (3).svg',
+    description:
+      'Personal portfolio that contains project information, skills, and contact',
+    desc1:
+      'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry.',
+    technologies: ['HTML', 'CSS', 'JavaScript'],
+    technologies1: ['github', 'ruby', 'Bootsrap'],
+    liveVersion: 'https://gmanlove.github.io/My-Portfolio/ ',
+    sourceCode: 'https://github.com/Gmanlove/My-Portfolio',
+  },
+];
+
+const workpage = document.querySelector('#workpage');
+for (let i = 0; i < projects.length; i += 1) {
+  workpage.innerHTML += `
+  <div class="card-proj">
+  <img src="${projects[i].img}" alt="project card image" />
+  <div class="project-card-iterms flex column">
+    <h2 class="card-title">${projects[i].name}</h2>
+    <div class="card-lists">
+      <p class="canopy">${projects[i].des[0]}</p>
+      <div class="round"></div>
+      <p>${projects[i].des[1]}</p>
+      <div class="round"></div>
+      <p>${projects[i].des[2]}</p>
+    </div>
+    <div class="card-note">
+      <p>
+      ${projects[i].description}
+      </p>
+    </div>
+    <div class="card-tag">
+      <ul class="card-skills flex">
+        <li>${projects[i].technologies[0]}</li>
+        <li>${projects[i].technologies[1]}</li>
+        <li>${projects[i].technologies[2]}</li>
+      </ul>
+    </div>
+    <button type="button" id="btn-${i}">See Project</button>
+  </div>
```

Comment:
NIce job making implemnting the changes requested by the previous reviewers👍,  I think your project cards currently matches the design on mobile, but on desktop it is still looking different from the given design😉. 

your project cards 
![Screenshot from 2023-02-02 10-04-04](https://user-images.githubusercontent.com/64643995/216281246-a560843e-460e-4214-8a10-7b0bc9413214.png)

currently an individual card takes almost the entire width and height of the screen, I think we can make some improvements on that, just take a look and [the design ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=fNRsCiKJ9Gq57Nqh-0)and make the comparison😉.

design
![image](https://user-images.githubusercontent.com/64643995/216282067-5416114d-2867-49fd-8185-764b5715b007.png)

kindly take a look at the spacings and make the necessary adjustments🤗
don't forget that it is a [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md) for this milestone that we stick to the [given design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=fNRsCiKJ9Gq57Nqh-0)